tree/30_generic_methods/file_copy_from_local_source.cf

```diff
@@ -28,6 +28,11 @@
 
 bundle agent file_copy_from_local_source(source, destination)
 {
+  vars:
+     "class_prefix"        string => canonify("file_copy_from_local_source_${destination}");
+     "new_class_prefix"    string => canonify(join("_", "this.callers_promisers"));
+
   methods:
-      "placeholder" usebundle => file_copy_from_local_source_recursion("${source}", "${destination}", "0");
+      "copy without recursion" usebundle => file_copy_from_local_source_recursion("${source}", "${destination}", "0");
+      "new result classes"     usebundle => _classes_copy("${class_prefix}", "${new_class_prefix}");
```

Comment:
Oh yes, then we'll have to define the classes at the source, ie define both in a new body classes